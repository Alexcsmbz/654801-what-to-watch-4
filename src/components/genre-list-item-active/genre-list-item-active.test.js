import GenreListItemActive from './genre-list-item-active.tsx';

const genre = `genre`;

describe(`GenreListItemActive snapshot test`, () => {
  it(`GenreListItemActive should render genre active`, () => {
    const tree = renderer.create(
        <GenreListItemActive
          item={genre}
          onClick={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
