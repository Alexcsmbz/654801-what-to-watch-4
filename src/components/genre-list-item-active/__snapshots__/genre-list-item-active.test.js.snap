// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreListItemActive snapshot test GenreListItemActive should render genre active 1`] = `
<li
  className="catalog__genres-item catalog__genres-item--active "
  onClick={[Function]}
>
  <div
    className="catalog__genres-link"
  >
    genre
  </div>
</li>
`;
